isis/src/base/apps/ringsautomos/ringsautomos.h

```diff
@@ -0,0 +1,11 @@
+#ifndef ringsautomos_h 
+#define ringsautomos_h
+
+#include "Cube.h"
```

Comment:
I forget the ISIS style guide on includes but this could probably just go in the cpp